app/components/inspect.js

```diff
@@ -0,0 +1,140 @@
+'use strict'
+
+import React from 'react'
+import styled from 'styled-components'
+import { toStr } from 'dat-encoding'
+import bytes from 'prettier-bytes'
+import FileList from './file-list'
+import icon from './icon'
+import { Plain as PlainButton } from './button'
+
+const DetailHeader = styled.header`
+    height: 4rem;
+    flex-shrink: 0;
+    border-bottom: 1px solid var(--color-neutral-20);
+`
+
+const DetailFooter = styled.footer`
+    flex-shrink: 0;
+    border-top: 1px solid var(--color-neutral-20);
+`
+
+const LabelStyle = styled.div`
+    font-size: .75rem;
+    min-width: 8rem;
+    color: var(--color-neutral-60);
+`
+
+const Label = ({ children }) => (
+  <LabelStyle className="mb2">
+    {children}
+  </LabelStyle>
+)
+
+const Inspect = ({ dat, closeInspectDat, addFilesToDat }) => {
+  if (!dat) return <div style={{ display: 'none' }}></div>
```

Comment:
similar to the comment on the `table` component, just return an empty `<Fragment />`